frontend/src/metabase/lib/api.js

```diff
@@ -179,20 +179,28 @@ export class Api extends EventEmitter {
           try {
             body = JSON.parse(body);
           } catch (e) {}
-          if (xhr.status >= 200 && xhr.status <= 299) {
+          let status = xhr.status;
+          if (
+            status === 202 &&
+            body._status !== null &&
```

Comment:
This is super minor but the idiomatic way to check for `null` or `undefined` in JS is `!= null` (the only reason you should ever use `==` or `!=` over `===` and `!==`)